src/main/java/uk/gov/hmcts/reform/managecase/security/SecurityUtils.java

```diff
@@ -52,4 +58,9 @@ private String removeBearerFromToken(String token) {
         return token.startsWith(BEARER) ? token.substring(BEARER.length()) : token;
     }
 
+    public boolean hasSolicitorRole(List<String> roles, String jurisdiction) {
+        return roles.stream().anyMatch(role -> SOLICITOR_ROLE.matcher(role).matches()
+            && role.toLowerCase(Locale.getDefault()).contains(jurisdiction.toLowerCase(Locale.getDefault())));
```

Comment:
@tomxelliott  does it match IAC requirement mentioned in the ACA-119?  (jurisdiction check)